engine/runtime/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -648,7 +810,13 @@ class SuggestionBuilderTest extends AnyWordSpecLike with Matchers {
               name       = "a",
               arguments = List(
                 Suggestion
-                  .Argument("this", "Unnamed.Test.MyType", false, false, None)
+                  .Argument(
+                    "this",
+                    "Unnamed.Test.MyType",
```

Comment:
What is the reason for this change? Some automatic refactoring? Otherwise the new construct looks semantically the same as the old one to me.